Fo fsphys 2017

Antujin opened the pull request:
Änderungen an der Fachschaftsordnung:

- Protokolle werden digital zugänglich gemacht (wie auch immer)

- Mitgliedschaft auf Verteiler wird immer nur für ein Jahr verlängert

- Erleichterungen für die Finanzverwaltung

Antujin committed:
digitale Veröffentlichung von Protokollen

Erzwingt digitale veröffetnlcihung von Sitzungsprotokollen (genaue Form bleibt offen)

Antujin committed:
Mitgliedschaft auf dem E-Mail Verteiler

Verlängerung für alte Mitglieder muss jährlich erneuert werden.

Antujin committed:
Anpassung der Daten und  Inkrafttreten

Antujin committed:
Erleichterung der Finanzverwaltung

So muss nciht abgestimmt werden ob N€ für Getränke und Fressalien vom AStA beantragt werden sollen (legalisiert den Status Quo)

l-althueser commented on docs/Fachschaftsordnung.tex:
```diff
@@ -149,6 +149,7 @@ \section{Arbeit des Fachschaftsrats}
 \begin{itemize}
 	\item Der FSR gewährleistet eine angemessene Beratung der Studierenden und sorgt für Präsenzzeiten. Die Präsenzzeiten werden in einem Plan festgehalten, der am Anfang eines jeden Semesters aufzustellen ist. In der Vorlesungszeit ist ein Minimum von zwei Tagen in der Woche mit insgesamt mindestens vier Stunden Präsenzzeit zu gewährleisten. Die Mitglieder des FSR sorgen für den Präsenzdienst. Währ ...
 	\item Der FSR sorgt durch Öffentlichkeitsarbeit für Transparenz und Anerkennung unter der Studierendenschaft. Dies wird unter anderem durch regelmäßige Information auf der Internetseite sowie durch Aushänge und Informationsveranstaltungen gewährleistet.
+	\item Protokolle über öffentliche Sitzungen des FSR werden zur Steigerung der Transparenz allen Physikstudierenden digital zugänglich gemacht
```
Das sollte entweder in anderer Form an die Zeile davor angehangen werden oder aber mit "Der FSR.." beginnen und dann nicht das Wort "Transparenz" enthalten.

l-althueser replied:
Und es fehlt ein Punkt am Ende :P

Antujin replied:
Passiert :p

l-althueser commented on docs/Fachschaftsordnung.tex:
```diff
@@ -179,7 +180,7 @@ \section{E-Mail-Verteiler und Nutzergruppe}
 	\item Der FSR unterhält einen E-Mail-Verteiler: \textbf{\email{fsphys-l@listserv.uni-muenster.de}}. Dieser wird von Herrn Dr.\ Adam (\email{adamh@uni-muenster.de}) technisch verwaltet. E-Mails an die E-Mail-Adresse der Fachschaft (\email{fsphys@uni-muenster.de}) werden vom Verwalter nach einer Spam-Filterung an den internen Verteiler weitergeleitet. Die interne Kommunikation des FSR erfolgt über ...
 	\item Mitglied in dem E-Mail-Verteiler wird jedes Mitglied des FSR. Dem Verwalter wird jeweils mitgeteilt, wie er über einen Aufnahmeantrag zu entscheiden hat.
-	\item Mit Austritt aus dem FSR endet die Mitgliedschaft im E-Mail-Verteiler. Alte Mitglieder können (mündlich) beantragen, weiterhin Mitglied im E-Mail-Verteiler zu bleiben. Der FSR entscheidet über den Antrag.
+	\item Mit Austritt aus dem FSR endet die Mitgliedschaft im E-Mail-Verteiler. Alte Mitglieder können (mündlich) beantragen, für ein weiteres Jahr Mitglied im E-Mail-Verteiler zu bleiben. Der FSR entscheidet über den Antrag.
```
Das würde insgesamt nur 1 Jahr bedeuten. Meintest du etwas wie "können für jeweils ein Jahr beantragen"?

Sollten wir den Terminus "Alte Mitglieder" in "ehemalige Mitglieder" überall umändern?

Antujin replied:
klingt nach nem Plan :D

l-althueser commented on docs/Fachschaftsordnung.tex:
```diff
@@ -195,4 +196,6 @@ \section{Fachschaftsvollversammlung}
 \section{Inkrafttreten}
 Die Fachschaftsordnung der Fachschaft Physik der Westfälischen Wilhelms Universität tritt durch Votum mit Zweidrittelmehrheit der Mitgliedern der Fachschaftsvertretung und durch öffentlichen Aushang am \formatdate{07}{01}{2016} in Kraft.
 
+Die erste Änderung der Fachschaftsordnung der Fachschaft Physik der WWU tritt ddurch Votum mit Zweidrittelmehrheit der Mitgliedern der Fachschaftsvertretung und durch öffentlichen Aushang am \formatdate{12}{1}{2017} in Kraft. Gleichzeitig tritt die Fachschaftsordnung vom \formatdate{6}{1}{2016} außer Kraft.
```
Rechtschreibfehler ..

Antujin committed:
Anmerkungen von Lutz

Die Kommentare von Lutz zu den letzten Commits wurden eingepflegt

Socob committed:
Kleinere/redaktionelle Änderungen

Socob reviewed (commented):
Bin damit einverstanden/habe keine wirklichen Anmerkungen, nur ein paar redaktionelle Korrekturen (s. 5885de97ce157b9dd90b4744f30cdf84c5c2902e).

Socob committed:
Präzisere Formulierung für öffentliche Protokolle vs. Sitzungen

l-althueser committed:
Update Fachschaftsordnung.tex

Socob committed:
Beschluss vom 2017-01-11

Fassung der Fachschaftsordnung, die auf der konstituierenden Sitzung der
Fachschaftsvertretung Physik am 2017-01-11 beschlossen wurde.
